### ISSUE TITLE
Escape both v and V

### ISSUE THREAD

**Issue by kaspersoerensen** [damienaicheh/azure-devops-mobile-versioning#13]
Hi there,
First off: Thank you for creating this extension, it is really helpful! 

Would it be possible to escape both capital V and non-capital v? It seems that if you have a git-tag called for instance ```V0.2.0``` the extension extracts ```V0``` as the ```$(MAJOR)```. If this is used to update the ```bundleShortVersionString``` in iOS the build fails to upload, because ```bundleShortVersionString``` can only contain integers: [docs](https://developer.apple.com/documentation/bundleresources/information_property_list/cfbundleshortversionstring)

Maybe it is a matter of updating: https://github.com/damienaicheh/azure-devops-mobile-versioning/blob/24ff6770a3bc13e5a61b8c7bc215717b2c58df25/tasks/ExtractVersionFromTagTask/extract-version-from-tag.js#L47

and

https://github.com/damienaicheh/azure-devops-mobile-versioning/blob/24ff6770a3bc13e5a61b8c7bc215717b2c58df25/tasks/ExtractVersionFromTagTask/extract-version-from-tag.ts#L47

and add something along the lines of:
``` javascript
if (tag.includes('v') || tag.includes('V') ) {
  var tagSplitted = tag.toLowerCase().split('v');
  tag = tagSplitted[1];
}
```

P.S.: my ```.js``` or ```.ts``` skills are non-existing - so there might very well be a better way to solve this.

**Comment by damienaicheh** [comment]
Hi, 

Thanks for your feedback.
I can change the test by transforming all the tag to lowercase before splitting it. So it will handle all the case.

I don't know when I will do it but for a next version why not.

**Comment by damienaicheh** [comment]
Available on version 1.5.0 of the task and on the 3.2.0 of the global package.
